src/components/views/rooms/Autocomplete.js

```diff
@@ -100,11 +101,26 @@ export default class Autocomplete extends React.Component {
         this.setState({selectionOffset});
     }
 
+    componentDidUpdate() {
+        // this is the selected completion, so scroll it into view if needed
+        const selectedCompletion = this.refs[`completion${this.state.selectionOffset}`];
+        if (selectedCompletion && this.container) {
+            const {offsetTop} = ReactDOM.findDOMNode(selectedCompletion);
```

Comment:
To me, this syntax feels a bit unnecessary and less clear than `const offsetTop = foo.offsetTop`, but that's just a personal thing.